.circleci/config.yml

```diff
@@ -4,160 +4,24 @@
 version: 2
 jobs:
 
-  build_2.3.4_rails_3:
+  build_2.5_rails_5:
```

Comment:
If we arent going to use the erb to generate this file anymore then I suggest we delete it.